docs/fides/docs/language/resources/system.md

```diff
@@ -9,12 +9,11 @@ A System is a model for describing anything that processes data for your organiz
             |-> privacy declarations
   ```
 
-
 ## Object Structure
 
-**fides_key**<span class="required"/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;_string_
+**fides_key**<span class="required"/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;_constrained string_
 
-A string token of your own invention that uniquely identifies this System. It's your responsibility to ensure that the value is unique across all of your System objects. The value may only contain alphanumeric characters and underbars (`[A-Za-z0-9_]`). 
+A string token of your own invention that uniquely identifies this System. It's your responsibility to ensure that the value is unique across all of your System objects. The value may only contain alphanumeric characters and underbars (`[A-Za-z0-9_.-]`).
```

Comment:
```suggestion
A custom string token that uniquely identifies this System. It must be unique across all System objects. The value may only contain alphanumeric characters, `_`, `.`, and `-` (it will be evaluated against the regular expression: `[A-z0-9_.-]`).
```